### PR TITLE
PR for added travis ci doc

### PR DESCRIPTION
**Pull request by vanetten**
### Documentation changes

Added documentation for configuring Travis CI.

### CODE REVIEW

File: doc/dictionary-travis-ci.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# TL;DR
+Configure continuous integration (test, build, deploy) of a Gen3 dictionary with Travis CI.
+## Prerequisites
+* Public dictionary repository (e.g. [bioteam/dictionaryutils](https://github.com/bioteam/dictionaryutils.git))
```
Comment (paulineribeyre) [inline doc/dictionary-travis-ci.md:4]: this repo doesn't seem to be public, let's replace the link with one of our dictionaries, maybe https://github.com/uc-cdis/canine_dictionary

File: doc/dictionary-travis-ci.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* Public S3 bucket ([how-to](https://www.simplified.guide/aws/create-public-s3-bucket))
+* Travis CI account ([sign up free for public repos](https://travis-ci.com/))
+* Development Environment ([for Mac](#development-environment))
+## Configure Dictionary
+### Clone dictionary repository
+```bash
+git clone https://github.com/bioteam/dictionaryutils.git
```
Comment (paulineribeyre) [inline doc/dictionary-travis-ci.md:11]: same here

File: doc/dictionary-travis-ci.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+4. Tag
+5. Push
+6. Observe
+7. Verify
+```bash
+testdict
+git commit -am "tagged release 1.0.0"
+git tag -a 1.0.0
+git push origin master --follow-tags
+open https://travis-ci.com/github/bioteam/dictionaryutils
```
Comment (paulineribeyre) [inline doc/dictionary-travis-ci.md:45]: same here

File: doc/dictionary-travis-ci.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+open https://bms-gen3-dev.s3.amazonaws.com/datadictionary/1.0.0/schema.json
```
Comment (paulineribeyre) [inline doc/dictionary-travis-ci.md:46]:
```suggestion
open https://<your-public-s3-bucket>/<folder-within-s3-bucket>/<branch or tag>/schema.json
```

File: doc/dictionary-travis-ci.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+brew install pyenv
+pyenv install 3.6.10
+echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.zshrc
+```
+### Install travis
+```bash
+brew install travis
+```
+### Install Docker ([Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac/))
+### Install tesdict
```
Comment (paulineribeyre) [inline doc/dictionary-travis-ci.md:69]:
```suggestion
### Install testdict
```

File: doc/dictionary-travis-ci.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+```
+### Install python 3.6
+```bash
+brew install pyenv
+pyenv install 3.6.10
+echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.zshrc
+```
+### Install travis
+```bash
```
Comment (paulineribeyre) [inline doc/dictionary-travis-ci.md:65]:
```suggestion
Install the travis CLI: https://github.com/travis-ci/travis.rb#installation
```bash
```

File: doc/dictionary-travis-ci.md (hunk 1 above)
Comment (vzpgb) [reply doc/dictionary-travis-ci.md:4]: I can see bioteam repo without logging in. I agree that canine dictionary is also a good example.

Comment (paulineribeyre) [reply doc/dictionary-travis-ci.md:4]: yeah i can also see it now. i would still rather link to a dictionary we own, and for now the demo dictionary is canine

**Pull request closed without merging**